src/Event.hpp

```diff
@@ -2,12 +2,42 @@
 #define WARPED_EVENT_HPP
 
 #include <string>
+#include <bitset>
 #include "serialization.hpp"
 
 namespace warped {
+    
+static constexpr std::uint8_t bitset_size {194U};
 
 struct compareEvents;
 
+template<size_t sRes,size_t s1,size_t s2,size_t s3, size_t s4, size_t s5>
+std::bitset<sRes> concatLoop( 
```

Comment:
reason why i went with this concatination, its faster.
https://stackoverflow.com/questions/3061721/concatenate-boostdynamic-bitset-or-stdbitset